code/style.css

```diff
@@ -1,7 +1,261 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  width: auto;
+  font-family: 'Poppins', sans-serif;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: 1fr;
+  justify-content: center;
+  /* padding: 1rem; */
+}
+
+/* ----------- HEADER ----------- */
+
+header {
+  display: flex;
+  justify-content: center;
+  background-image: url(./images/plate.jpeg);
+  background-size: cover;
+  background-position: center;
+  width: 100%;
+  object-fit: cover;
+  min-height: 50vh;
+  padding: 0;
+  margin: 0;
+}
+
+.header-title {
+  color: rgb(0,51,51, 0.75);
+  padding: 1rem 0rem;
+  font-size: 50px;
+  font-weight: 800;
+  text-align: center;
+  align-self: center;
+  text-transform: uppercase;
+}
+
+/* ----------- SIGN-UP SECTION ----------- */
+
+form {
+  display: grid;
+  justify-content: center;
+  gap: 1rem;
+  background-color: rgb(0,51,51);
+  border: 5px solid rgb(0,51,51);
+  padding: 1rem 0rem;
+}
+
+.form-header {
+  display: flex;
+  justify-content: center;
+  color: white;
+}
+
+label {
+  color: white;
+  font-size: 14px;
+}
+
+input {
+  margin: 0.25rem 0rem;
+  padding: 0;
+}
+
+input[type="text"], input[type="email"] {
+  width: 100%;
+  padding: 0.5rem 0.25rem;
+}
+
+input[type="radio"] {
+  margin-right: 0.25rem;
+}
+
+.radio-group {
+  display: flex;
+  justify-content: space-around;
+}
+
+.preference {
+  padding-right: 1rem;
+}
+
+.preference-option {
```

Comment:
Text color inside the scroll down menu is set to blue as default in iOS.
Solve it by setting a color in the button tag (I think).

Reply:
Thanks for pointing this out! I didn't notice it until after deploying, at which point I had no time to change it. I will look into it, and a tip to myself ns to deploy earlier to check iOS before it's too late!